source/fem-dem/fluid_dynamics_vans_matrix_free_operators.cc

```diff
@@ -53,43 +53,55 @@ VANSOperator<dim, number>::evaluate_non_linear_term_and_calculate_tau(
 
 template <int dim, typename number>
 void
-VANSOperator<dim, number>::evaluate_void_fraction(
-  const VoidFractionBase<dim> &void_fraction_manager)
+VANSOperator<dim, number>::compute_void_fraction(
+  const LinearAlgebra::distributed::Vector<double> &void_fraction_solution,
+  const DoFHandler<dim>                            &void_fraction_dof_handler)
 {
-  AssertThrow(
-    void_fraction_manager.void_fraction_parameters->mode ==
-      Parameters::VoidFractionMode::function,
-    ExcMessage(
-      "The VANS matrix free solver only supports void fractions established using a function. Void fraction derived from functions will be developed in a future version of Lethe."));
-
-  this->timer.enter_subsection("operator::evaluate_void_fraction");
+  this->timer.enter_subsection("operator::compute_void_fraction");
 
   const unsigned int n_cells = this->matrix_free.n_cell_batches();
```

Comment:
Why only n_cell_batches and not `matrix_free.n_cell_batches() + matrix_free.n_ghost_cell_batches()`?